Makefile

```diff
@@ -78,6 +78,30 @@ update_autocomplete:
 	curl -sL https://github.com/openshift/origin/raw/master/contrib/completions/bash/oadm > bash_completions/oadm
 	curl -sL https://github.com/openshift/origin/raw/master/contrib/completions/bash/oc > bash_completions/oc
 	curl -sL https://github.com/openshift/origin/raw/master/contrib/completions/bash/openshift > bash_completions/openshift
+	
+update_template:
+	# Clone from upstream and put required location.
+	# Nodejs
+	curl -sL https://github.com/openshift/nodejs-ex/raw/master/openshift/templates/nodejs-mongodb.json > services/openshift/templates/common/nodejs-mongodb.json
+	curl -sL https://github.com/openshift/nodejs-ex/raw/master/openshift/templates/nodejs.json > services/openshift/templates/common/nodejs.json
+	# Cakephp
+	curl -sL https://github.com/openshift/cakephp-ex/raw/master/openshift/templates/cakephp.json > services/openshift/templates/common/cakephp.json
+	curl -sL https://github.com/openshift/cakephp-ex/raw/master/openshift/templates/cakephp-mysql.json > services/openshift/templates/common/cakephp-mysql.json
+	# Jenkins
+	curl -sL https://github.com/openshift/origin/blob/master/examples/jenkins/pipeline/jenkinstemplate.json > services/openshift/templates/adb/jenkinstemplate.json
```

Comment:
@praveenkumar please rename file _services/openshift/templates/adb/**jenkinstemplate**.json_ to _services/openshift/templates/adb/**jenkins-template**.json_ to ensure naming consistency. I am aware that upstream is inconsistent in this matter.

(There can't be 2 templates with same name, so naming consistency is especially important here. Say someone wants to customize Vagrantfile like [here](https://github.com/redhat-kontinuity/adb-openshift-runtime/blob/master/Vagrantfile#L80)...)